index.js

```diff
@@ -4,32 +4,236 @@ const logo = document.querySelector('#logo');
+        para2: 'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+
+        language: {
+          li1: 'HTML',
+          li2: 'Css',
+          li3: 'JavaScript',
+          li4: 'bootstrap',
+          li5: 'query',
+          li6: 'react.js',
+        },
+      },
+
+      {
+        id: 3,
+        key: 'three3',
+        heading: 'Tonic',
+
+        frame: { li1: 'CANOPY', li2: 'Back End Dev', li3: '2019' },
+
+        img: 'images/Snapshoot3.png',
+        alt: 'Tonic project',
+        para1: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+        para2: 'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+
+        language: {
+          li1: 'HTML',
+          li2: 'Css',
+          li3: 'JavaScript',
+          li4: 'bootstrap',
+          li5: 'query',
+          li6: 'react.js',
+        },
+      },
+
+      {
+        id: 4,
+        key: 'four4',
+        heading: 'Gateway',
+
+        frame: { li1: 'CANOPY', li2: 'Back End Dev', li3: '2019' },
+
+        img: 'images/Snapshoot4.png',
+        alt: 'gateway project',
+        para1: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+        para2: 'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+
+        language: {
+          li1: 'HTML',
+          li2: 'Css',
+          li3: 'JavaScript',
+          li4: 'bootstrap',
+          li5: 'query',
+          li6: 'react.js',
+        },
+      },
+    ];
+
+    projects.forEach((value, index) => {
+      // console.log(value.key , index)
```

Comment:
- [ ] Kindly consider removing the commented code as it makes our codebase look messy.